# Generated by Django 2.0.7 on 2019-06-25 08:12

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('submissions', '0044_exercise_not_found_error'),
    ]

    operations = [
        migrations.AddField(
            model_name='course',
            name='data_url',
            field=models.URLField(null=True),
        ),
    ]
